web-app/CKS.Web/Startup.cs

```diff
@@ -39,6 +39,8 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
             }
 
+            app.UseStatusCodePagesWithReExecute("/{0}.html");
+
 			app.UseDefaultFiles();
 			app.UseStaticFiles();
```

Comment:
We'll need to do the following here to get the correct status code:

```suggestion
			app.UseStaticFiles(new StaticFileOptions { OnPrepareResponse = context => {
                if(context.Context.Request.Path == "/404.html")
                        context.Context.Response.StatusCode = 404;
            } });
```

Reply:
done